gpsd_format/drivers.py

```diff
@@ -214,24 +201,39 @@ def read(self, size):
         return self.obj.read(size)
 
 
-class NewlineJSON(BaseDriver):
+class FileDriver(BaseDriver):
```

Comment:
It's the other way around. BaseDriver now has less code and does not do the open() call. FileDriver works like the old BaseDriver.